### PR TITLE
chore/code style changes to common format

### PR DESCRIPTION
**Pull request by potapovDim**
**Thanks for contributing to Selenium!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

nodejs selenium-webdriver lib code style changes to common format

### Description
nodejs selenium-webdriver lib code style changes to common format

### Motivation and Context
Align code style to common format 

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [ ] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: javascript/node/selenium-webdriver/lib/virtual_authenticator.js [hunk 1]
```diff
@@ -22,27 +22,26 @@
  * @enum {string}
  */
 const Protocol = {
-  'CTAP2': 'ctap2',
```
Comment (diemol) [inline javascript/node/selenium-webdriver/lib/virtual_authenticator.js:25]: Some tests use this, like `virtualAuthenticator_test.js`, so this is probably why CI is not passing. Can you please double check?

Same for `Transport` below.

**Review by diemol — DISMISSED** [review]

File: javascript/node/selenium-webdriver/lib/virtual_authenticator.js (hunk 1 above)
Comment (potapovDim) [reply javascript/node/selenium-webdriver/lib/virtual_authenticator.js:25]: @diemol yes sure, let me check that

Comment (potapovDim) [reply javascript/node/selenium-webdriver/lib/virtual_authenticator.js:25]: @diemol i checked tests locally and everything is fine 
```../../../bazel-bin/javascript/node/selenium-webdriver/lib/atoms/get-attribute.js
../../../bazel-bin/javascript/node/selenium-webdriver/lib/atoms/is-displayed.js
../../../bazel-bin/javascript/node/selenium-webdriver/lib/atoms/find-elements.js
[INFO] Using Selenium remote end: http://localhost:4444/wd/hub/
[INFO] Running tests against [chrome]


  [chrome]
    VirtualAuthenticator Test Suit 2
      ✓ should test create authenticator (352ms)
      ✓ should test remove authenticator
      ✓ should test add non-resident credential (220ms)
      ✓ should test add non-resident credential when authenticator uses U2F protocol (220ms)
      ✓ should test add resident credential (231ms)
      ✓ should test add resident credential not supported when authenticator uses U2F protocol
      ✓ should test get credentials (349ms)
      ✓ should test remove credential by rawID (352ms)
      ✓ should test remove credential by base64url Id (348ms)
      ✓ should test remove all credentials (467ms)
      ✓ should test set user verified (465ms)


  11 passing (21s)```

**Comment by potapovDim** [comment]
@harsha509 @diemol  hey guys, can you please clarify few moments related to nodejs ? 
we have two test runners in selenium-webdriver - mocha and jasmine, can we use one, or why we should use both  ?
we have many variables declaration by `var` keyword - do we want to move fully to `let/const`?
we have `class Select` and it has async call in constructor which allows me to use this class only with classic select lists, but not with some customs select list with same behavior but another tags, can we for example create another library i.e selenium-webdriver-extensions with classes like `Select`?
we have prettier and lint, but seems like we do not use it 100% ?

**Comment by diemol** [comment]
> @harsha509 @diemol hey guys, can you please clarify few moments related to nodejs ? we have two test runners in selenium-webdriver - mocha and jasmine, can we use one, or why we should use both ? we have many variables declaration by `var` keyword - do we want to move fully to `let/const`? we have `class Select` and it has async call in constructor which allows me to use this class only with classic select lists, but not with some customs select list with same behavior but another tags, can we for example create another library i.e selenium-webdriver-extensions with classes like `Select`? we have prettier and lint, but seems like we do not use it 100% ?

I'd be easier if you open issues/feature requests for each item and then we can track each one of them separately. Let's complete this PR and get it merged.

File: javascript/node/selenium-webdriver/test/devtools_test.js [hunk 2]
```diff
@@ -35,15 +35,18 @@ suite(
...
     ignore(browsers(Browser.CHROME)).it(
-      'sends Page.enable command using devtools', async function () {
+      'sends Page.enable command using devtools',
+      async function () {
         const cdpConnection = await driver.createCDPConnection('page')
         cdpConnection.execute('Page.enable', {}, function (_res, err) {
           assert(!err)
         })
-      })
+      }
```
Comment (harsha509) [inline javascript/node/selenium-webdriver/test/devtools_test.js:44]: HI @potapovDim,

Changes looks good!

But in few places I see this `})` is moved to multi-line

`})` is more readable I think!

Comment (potapovDim) [reply javascript/node/selenium-webdriver/test/devtools_test.js:44]: @harsha509 those changes were done based on prettier and eslint codestyle formats, i can update it, but wanted to follow formats

File: javascript/node/selenium-webdriver/io/exec.js [hunk 3]
```diff
@@ -158,6 +158,8 @@ module.exports = function exec(command, opt_options) {
 
 // Exported to improve generated API documentation.
 
-module.exports.Command = Command
-module.exports.Options = Options
-module.exports.Result = Result
+module.exports = {
```
Comment (harsha509) [inline javascript/node/selenium-webdriver/io/exec.js]: looks like there is an issue in exporting modules this way!

Comment (harsha509) [reply javascript/node/selenium-webdriver/io/exec.js]: @potapovDim can you recheck!

Comment (harsha509) [reply javascript/node/selenium-webdriver/io/exec.js]: This commit https://github.com/SeleniumHQ/selenium/commit/f4ce77d24ee364279326c87a5ddc8a3ea386b88b should fix and run tests correctly!

Comment (potapovDim) [reply javascript/node/selenium-webdriver/io/exec.js]: @harsha509 you are right! my bad, thanks for fix !

**Review by harsha509 — APPROVED** [review]
Thank you @potapovDim!

**Pull request merged**